bin/filter-json

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+// Read JSON from stdin, keep keys passed as arguments, write the result to stdout as JSON
+
+let args = process.argv.slice(2);
+let data = [];
+
+process.stdin.on("data", chunk => {
```

Comment:
Since `extract-sourcemap` already both finds and decodes the source map, we could also tack this filtering onto the end there.
That way we could avoid the streaming gymnastics.